### ISSUE TITLE
Make the Seek in AsEnumerable check CanSeek

### ISSUE THREAD

**Issue by dbeattie71** [Cinchoo/ChoETL#52]
I'm using a `ChoCSVReader` over a `ZipInputStream` and it fails because `ZipInputStream` isn't seekable.

Any harm in doing something like:
```csharp
if( sr is StreamReader sr2 && sr2.BaseStream.CanSeek )
                sr2.Seek( 0, SeekOrigin.Begin );
```

**Comment by Cinchoo** [comment]
Put a fix, pushed to nuget as 1.1.0.5-alpha3

Try it and let me know.

**Comment by dbeattie71** [comment]
That does it, thanks!
